### PR TITLE
Plushies!

### PR DESCRIPTION
**Pull request by juniwoofs**
# Description

Added two new plushies to the game! A harpy plushie and a plushie for the beloved station pet Morty!

image
mortplush
harpyplushie

---

# Changelog

:cl:
- add: two new cuddly friends to the station! (harpy and morty plush)

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
Fari_awe_2

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -1966,3 +1966,63 @@
...
+  - type: StaminaDamageOnHit
+    damage: 0.8
+  - type: EmitSoundOnActivate
+    sound:
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
+      params:
+        variation: 0.05
+  - type: EmitSoundOnUse
+    sound:
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:1998]: swap the \ to a /

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 2]
```diff
@@ -1966,3 +1966,63 @@
...
+  - type: EmitSoundOnCollide
+    sound:
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:2001]: and here

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 3]
```diff
@@ -1966,3 +1966,63 @@
...
+      params:
+        variation: 0.05
+  - type: EmitSoundOnLand
+    sound: 
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:2006]: and here

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 4]
```diff
@@ -1966,3 +1966,63 @@
...
+  - type: UseDelay
+    delay: 0.8
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 0.5
+    damage:
+      types:
+        Blunt: 0
+    soundHit:
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:2018]: and here

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 5]
```diff
@@ -1966,3 +1966,63 @@
...
+      params:
+        variation: 0.05
+    soundSwing:
+      path: /Audio\DeltaV\Voice\Harpy\caw1.ogg
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:2022]: you get the point

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 6]
```diff
@@ -1966,3 +1966,63 @@
     - type: Sprite
       sprite: Objects/Fun/toys.rsi
       state: shadowkin
+
+- type: entity
+  parent: BasePlushie
+  id: PlushieMort
+  name: Morty plushie
```
Comment (sleepyyapril) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:1973]: lowercase

**Review by sleepyyapril — CHANGES_REQUESTED** [review]

**Comment by VMSolidus** [comment]
Oh yea this also needs to add the two new plushies to the relevant plushy spawners.

**Comment by juniwoofs** [comment]
how would i go about doing that?

**Comment by VMSolidus** [comment]
image

Add the two plushies to this list to make them appear ingame. :)

**Review by sleepyyapril — APPROVED** [review]

**Comment by juniwoofs** [comment]
oop april did it for me ehehehe

**Pull request merged**